Mage.Sets/src/mage/cards/s/StriderRangerOfTheNorth.java

```diff
@@ -84,11 +84,10 @@ public boolean apply(Game game, Ability source) {
         game.addEffect(effect, source);
 
         // Checks if the creature's power is 4 or greater and then add First Strike until end of turn
-        if (permanent.getPower().getValue() >= 4){
+        if (permanent.getPower().getValue() + 1  >= 4) {
```

Comment:
This power check is not taking into account the temporary `+1/+1` attributed earlier to the creature so when a `3/3` creature is chosen and gets the `+1/+1` it is not receiving `first strike` although it is technically a `4/4` until end of turn.

Any way to get around this without just adding a `+1` to the if condition?

Reply:
Try using common effect classes instead of a custom effect. References: Dormant Grove, Tail Swipe. Please confirm testing Ent's Fury for this sort of thing as well.

Reply:
Savage Swipe appears to use `game.getState().processAction(game);` to apply one effect before proceeding to the next within the `apply()` method of the custom class. Still, common classes are preferred to streamline the codebase.

Reply:
Not sure why that way wouldn't work, but this implementation is fine